singer-connectors/tap-fastly/requirements.txt

```diff
@@ -0,0 +1,2 @@
+#pipelinewise-tap-s3-csv==1.1.0
+-e /Users/aidanwall/projects/analytics-pipeline/tap-fastly
```

Comment:
you need to github instead of your home directory (and remove the comment line above).